utils/loaddata_utils.py

```diff
@@ -69,7 +93,7 @@ def edit_loaddata_csv(path_to_loaddata_csv:pathlib.Path):
     """
     loaddata_df = pd.read_csv(path_to_loaddata_csv)
 
-    # finds the last z-plane value and assigns it as a variable 
+    # finds the last z-plane value and assigns it as a variable
     # Metadata_PlaneID values are 1-3 which correlates to the file names p01-p03
     final_z = max(loaddata_df["Metadata_PlaneID"].unique())
```

Comment:
Will this always work? If certain wells of a plate had different channel numbers, it might not. Probably ok for now, just food for thought!

Reply:
Based on my experience with the Interstellar data only, I don't think there should be different number of z-stacks within the same folder of images. I would assume that if not all of the images are the same number of z-plane and channels, there would be a problem.

Thank you for bringing this up though and I will keep my eye for it in the future! I will edit the code comment to be less hard coded.